src/NetscriptFunctions/Gang.ts

```diff
@@ -297,6 +297,22 @@ export function NetscriptGang(): InternalAPI<IGang> {
         ...member.getAscensionResults(),
       };
     },
+    getInstallResult: (ctx) => (_memberName) => {
+      const memberName = helpers.string(ctx, "memberName", _memberName);
+      getGang(ctx);
```

Comment:
What is the purpose of this, you don't use the result?

Reply:
With `getGang()` it's just doing the same thing all the other functions do, that method throws if the player doesn't have a gang. And same goes for the method names, I just did the exact same thing `getAscensionResult()` does, plural in the gang member function and singular in the player exposed function.

Reply:
Should I add that comment to every function then?

Reply:
Ah I see, it is actually a common pattern among several of the other functions. I guess nothing needs to change then.